autogpt_platform/backend/backend/data/credit.py

```diff
@@ -18,7 +18,7 @@
     LlmModel,
 )
 from backend.blocks.talking_head import CreateTalkingAvatarVideoBlock
-from backend.data.block import Block, BlockInput
+from backend.data.block import Block, BlockInput, get_block
```

Comment:
We can't pass non-builtin non-pydantic object on param & return value on Pyro request, so this needs to be changed.

Reply:
yes that's what I meant